stanford_image.features.field_instance.inc

```diff
@@ -27,94 +27,256 @@ function stanford_image_field_default_field_instances() {
       'stanford_3_col_header' => array(
         'label' => 'hidden',
         'module' => 'text',
-        'settings' => array(),
+        'settings' => array(
```

Comment:
We're not using custom_formatters in JS*, are we? @cynmij did you have that module installed on the site that you exported this Feature from?